test/test.cpp

```diff
@@ -163,3 +173,25 @@ TEST(Clip, Points) {
     ASSERT_EQ(expected1, clipped1);
     ASSERT_EQ(expected2, clipped2);
 }
+
+TEST(GetTile, USStates) {
+    const auto geojson = mapbox::geojson::parse(loadFile("test/fixtures/us-states.json"));
+    GeoJSONVT index{ geojson.get<mapbox::geojson::feature_collection>() };
+
+    const auto features = index.getTile(7, 37, 48).features;
+    const auto expected = parseJSONTile(loadFile("test/fixtures/us-states-z7-37-48.json"));
+    ASSERT_EQ(features == expected, true);
+
+    const auto square = parseJSONTile(loadFile("test/fixtures/us-states-square.json"));
+    ASSERT_EQ(square == index.getTile(9, 148, 192).features, true);  // clipped square
+    ASSERT_EQ(square == index.getTile(11, 592, 768).features, true); // clipped square
+
+    ASSERT_EQ(empty_tile == index.getTile(11, 800, 400), true);  // non-existing tile
+    ASSERT_EQ(&empty_tile == &index.getTile(11, 800, 400), true); // non-existing tile
+
```

Comment:
replicated the error: https://gist.github.com/springmeyer/445c1d2d143e59c5d340b2d053f67008. Something is weird with the comparison operator, but I'm not able to see it yet. Wondering if @jfirebaugh might be able to spot it...